sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/StorageTestEnvironment.cs

```diff
@@ -16,40 +16,81 @@ namespace Azure.Messaging.EventHubs.Processor.Tests
     public static class StorageTestEnvironment
     {
         /// <summary>The active Azure storage connection for this test run, lazily created.</summary>
-        private static readonly Lazy<StorageScope.StorageProperties> s_activeStorageAccount =
-            new Lazy<StorageScope.StorageProperties>(CreateStorageAccount, LazyThreadSafetyMode.ExecutionAndPublication);
+        private static readonly Lazy<StorageScope.StorageProperties> ActiveStorageAccount =
+            new Lazy<StorageScope.StorageProperties>(CreateStorageAccountIfMissing, LazyThreadSafetyMode.ExecutionAndPublication);
+
+        /// <summary>The environment variable value for the storage account connection string, lazily evaluated.</summary>
+        private static readonly Lazy<string> ActiveStorageAccountConnectionString =
+            new Lazy<string>(() => ReadEnvironmentVariable("EVENT_HUBS_BLOBS_CONNECTION_STRING"), LazyThreadSafetyMode.PublicationOnly);
 
         /// <summary>
         ///   Indicates whether or not an ephemeral storage account was created for the current test execution.
         /// </summary>
         ///
         /// <value><c>true</c> if an Azure storage account was created; otherwise, <c>false</c>.</value>
         ///
-        public static bool WasStorageAccountCreated => s_activeStorageAccount.IsValueCreated;
+        public static bool WasStorageAccountCreated => ActiveStorageAccount.IsValueCreated;
 
         /// <summary>
         ///   The name of the Azure storage account to be used for Live tests.
         /// </summary>
         ///
         /// <value>The name will be determined by creating an ephemeral Azure storage account for the test execution.</value>
         ///
-        public static string StorageAccountName => s_activeStorageAccount.Value.Name;
+        public static string StorageAccountName => ActiveStorageAccount.Value.Name;
 
         /// <summary>
         ///   The connection string for the Azure storage instance to be used for Live tests.
         /// </summary>
         ///
         /// <value>The connection string will be determined by creating an ephemeral Azure storage account for the test execution.</value>
         ///
-        public static string StorageConnectionString => s_activeStorageAccount.Value.ConnectionString;
+        public static string StorageConnectionString => ActiveStorageAccount.Value.ConnectionString;
+
+        /// <summary>
+        ///   It tries to read the EVENT_HUBS_BLOBS_CONNECTION_STRING environment variable.
+        ///   If not found, it creates a new storage account on Azure.
+        /// </summary>
+        ///
+        /// <returns>The active Azure storage account for this test run.</returns>
+        ///
+        private static StorageScope.StorageProperties CreateStorageAccountIfMissing()
```

Comment:
Not sure that the name here gives us context for "we'll try to use one and then create one if we don't have it."    I'm trying to think of a name that isn't crazy long and does that, but naming is hard.  😄 

Maybe `EnsureStorageAccount` with the comments calling out the semantics of "if you give us one, we'll use it otherwise we'll create one" like you're already doing?